pages/index.js

```diff
@@ -35,14 +39,78 @@ const Landing = () => {
       </Head>
       <Container>
         <Main>
-          <LogoText src={HackSC} alt="HackSC" />
-          <Header>Out for lunch.</Header>
-          <Subtitle>Returning with more news this fall.</Subtitle>
-          <Links>
-            <Button href="https://2019.hacksc.com">
-              See site for HackSC 2019
-            </Button>
-          </Links>
+          <Header>
+            <TitleContainer>
+              {/* want the 2020 to be stacked like:
+                        20
+                        20
+              to the right of the HackSC*/}
+              <TwentyRight>2020</TwentyRight>
+              <LogoText src={HackSC} alt="HackSC" />
+              <HeaderTitle>JAN 31, 2020 - FEB 2, 2020</HeaderTitle>
+              <HeaderSubtitle>LOS ANGELES, CALIFORNIA</HeaderSubtitle>
+            </TitleContainer>
+
+            <Links>{/* <Button>
```

Comment:
same thing, we'll prob want to comment this out and add a TODO comment (just to keep track of things we want to come back to)